### DISCUSSION TITLE
Column CSS

### DISCUSSION THREAD

**Discussion by EldoonNemar** [Okipa/laravel-table#121] [Q&A] [answered]
Is there a way to define column css? For example, ```d-none d-lg-table-cell``` to hide columns when the display is small, as is native with bootstrap 4+.

**Comment by Okipa** [comment, marked as answer]
Hi @HawtDogFlvrWtr there is no such thing yet but it would be a great idea in order to improve responsiveness of the tables.

Would you like to make a PR with this addition ?

**Comment by EldoonNemar** [comment]
yup. I can do that when I have a few cycles. Thanks for the quick response.
